### DISCUSSION TITLE
Enregistrement titre restaurant

### DISCUSSION THREAD

**Discussion by sylvestredavid** [PayGreen/documentation#21] [Q&A] [answered]
Bonjour,
pour notre plateforme, nous voudrions que les users puissent enregistrer leurs titres restaurants, en sandbox nous n'avons aucun soucis car la boutique de base du compte accepte bien les titres restaurants (conecs, swile et restoflash), mais en production, les formulaires s'ouvrent et se ferment directement... après avoir vérifier sur le dashboard, je pense que cela vient du fait que la boutique de base du compte n'accepte pas le titres restaurants... et quand je fais la demande pour les accepter c'est refusé.
Comment puis-je palier à ça?
Merci d'avance pour votre réponse.

**Comment by gaelDolle** [comment, marked as answer]
Bonjour,
Merci de contacter le support client en y indiquant les nom et identifiant de votre boutique. Cela vient peut-être de la configuration de votre boutique.
https://paygreen.atlassian.net/servicedesk/customer/portal/9
Cordialement
